src/app/components/DynamicProgress.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Icon } from 'react-components';
+import contactLoader from 'design-system/assets/img/pm-images/contact-export-loader.gif';
+
+const DyanmicProgress = ({ id, alt, displayDuring, displayEnd, value, max, ...rest }) => {
+    return (
+        <div className="aligncenter">
+            {value < max ? (
+                <img src={contactLoader} alt={`${alt}-loader`} />
+            ) : (
+                <Icon name="on" fill="success" size={100} alt={`${alt}-finished`} />
+            )}
+            <progress className="progress-contact w100 mt1" aria-describedby={id} value={value} max={max} {...rest} />
```

Comment:
Use `Progress` from `react-components` please.

Reply:
`value` and `max` are already set in `rest`.
```suggestion
            <Progress className="progress-contact w100 mt1" aria-describedby={id} {...rest} />
```